packages/frontend-core/src/components/grid/stores/ui.js

```diff
@@ -141,18 +146,18 @@ export const initialise = context => {
   })
 
   // Reset selected rows when selected cell changes
-  focusedCellId.subscribe(id => {
-    if (id) {
-      selectedRows.set({})
-    }
-  })
+  // focusedCellId.subscribe(id => {
+  //   if (id) {
```

Comment:
Should we remove the commented code here?

Reply:
Nice one, forgot to remove it after testing with them commented out!